### PR TITLE
Teke

### PR DESCRIPTION
**Pull request by ttddww**

### CODE REVIEW

**Comment by ttddww** [comment]
correct

**Review by chalewkg — COMMENTED** [review]
send to Sami for approval

**Review by chalewkg — COMMENTED** [review]
Send to Sami for final  review

File: backend/routes/employee.routes.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const authMiddleware = require("../middlewares/auth.middleware");
 // Create a route to handle the add employee request on post
 router.post(
   "/api/employee",
+  [authMiddleware.verifyToken, authMiddleware.isAdmin],
   employeeController.createEmployee
```
Comment (yasefa1) [inline backend/routes/employee.routes.js:13]: goog job

**Review by yasefa1 — APPROVED** [review]

**Pull request merged**